model/distro/db.go

```diff
@@ -163,6 +163,21 @@ func ByNeedsHostsPlanning(containerPools []evergreen.ContainerPool) db.Q {
 		}})
 }
 
+// ByIsDisabled returns a query that selects distros that don't run containers and are disabled
+func ByIsDisabled(containerPools []evergreen.ContainerPool) db.Q {
```

Comment:
This should probably only query for disabled distros that have a non-zero queue length so we don't repeatedly clear the queues on every scheduler populate job.

Reply:
Also, if you do that I think you can remove the container pools part of this, since container pools will have non-zero queue length.

Reply:
Do you mean that container pools will have zero queue length? My intent was to ignore pool distros because they are ignored by the planning functions above, but it would make sense if they always have zero queue length as a result of that.